src/load-config.js

```diff
@@ -0,0 +1,19 @@
+const process = require('process');
+const lodash = require('lodash');
+
+const requiredOptions = [
+    'name',
+    'config'
+];
+
+module.exports = (configFilePath) => {
+    const config = configFilePath ? require(`${process.cwd()}/${configFilePath}`) : {};
+    config.map((rack) => {
+        requiredOptions.map(option => {
+            if (!lodash.get(rack, option)) {
+                throw new Error(`${option} is required in config file`);
```

Comment:
this is too general. Assuming I have couple of racks in config and one of them is missing the property, this error says nothing about the place where the problem potentially exists. See suggestion.

```suggestion
                throw new Error(`${option} is required in the object ${JSON.stringify(rack)}`);
```